### PR TITLE
Add spec for uptime endpoints

### PR DESCRIPTION
**Pull request by mdanzinger**
This PR adds the specs for the uptime API

### CODE REVIEW

**Comment by mdanzinger** [comment]
!deploy

File: specification/resources/uptime/models/check.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      example: "https://www.landingpage.com"
+      description: The endpoint to perform healthchecks on.
+
+
+    regions:
+      type: array
+      items:
+        type: string
+        enum:
+          - us-east
```
Comment (mdanzinger) [inline specification/resources/uptime/models/check.yml:48]: TODO: Update these regions with our actual regions.

**Review by bsnyder788 — APPROVED** [review]
lgtm

**Review by Ashleycho35 — DISMISSED** [review]

**Comment by mdanzinger** [comment]
!deploy

**Comment by mdanzinger** [comment]
!deploy

**Comment by mdanzinger** [comment]
!deploy

**Comment by mdanzinger** [comment]
Uptime is now officially GA - this PR is ready for review.

**Review by ChiefMateStarbuck — APPROVED** [review]

File: specification/resources/uptime/delete_check.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+operationId: check_delete
+
+summary: Delete a Check
+
+description: |
+  To delete an Uptime check, send a DELETE request to `/v2/uptime/checks/$CHECK_ID`. A 204 status
+  code with no body will be returned in response to a successful request.
+
+
+  Deleting a check will also delete alerts assicoiated with the check.
```
Comment (prichrd) [inline specification/resources/uptime/delete_check.yml:10]:
```suggestion
  Deleting a check will also delete alerts associated with the check.
```

**Pull request merged**